packages/main/src/TimePicker.js

```diff
@@ -0,0 +1,640 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
```

Comment:
In RTL the "Hours" "Minutes" "Seconds" sliders are  mirrored "Seconds" "Minutes" "Hours". 
They remain unchanged in LTR RTL In openui5. This should be checked.

Reply:
Done